src/core/lib/security/security_connector/security_connector.cc

```diff
@@ -959,24 +959,29 @@ static grpc_slice compute_default_pem_root_certs_once(void) {
 }
 
 static grpc_slice default_pem_root_certs;
+static ssl_root_certs_store* default_root_store = nullptr;
 
-static void init_default_pem_root_certs(void) {
+static void init_default_ssl_root_store(void) {
   default_pem_root_certs = compute_default_pem_root_certs_once();
```

Comment:
this can be a local var unless I'm missing something.

Reply:
It can go away. When I first wrote this, I thought default_pem_root_certs may be needed for testing.